generate.py

```diff
@@ -51,6 +52,11 @@
         elif app_id == "com.android.vending":
             metadata["dependencies"] = ["com.google.android.gsf", "com.google.android.gms"]
 
+        if app_id in mirrored_google_apps:
+            metadata["source"] = "Google"
+        elif app_id[:15] in ["app.grapheneos.", "app.attestation"]:
```

Comment:
Change this to `elif app_id.startswith(("app.grapheneos.", "app.attestation."))` i.e. you can pass a tuple to `startswith`.

Reply:
resolved along with typo in google apps list